test/write_lockfile_test.rb

```diff
@@ -0,0 +1,239 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class WriteLockfileTest < Minitest::Test
```

Comment:
Is there an existing test that covers the case where the lockfile _shouldn't_ be rewritten?

Reply:
I added one to be sure as well as one where only the version number changes.